graphql/playground.ts

```diff
@@ -0,0 +1,207 @@
+import { ApolloServerPluginLandingPageGraphQLPlayground } from 'apollo-server-core';
+import { isDev } from '../common/util/environment';
+
+const userHeader = { authorization: 'Bearer <somerandombearer>' };
+const adminHeader = { authorization: "Basic <btoa('admin' + password)>" };
+
+const endpoint = '/apollo';
+
+const loginQuery = `mutation {
+   # Place some random Bearer token in the tab HTTP Headers, then log in using one of these mutations:
```

Comment:
Let's put the actual header as example, so that we can copy-paste it & just replace the token.

Reply:
The header is already prefilled :)

Reply:
Unfortunately we cannot generate a random token as the tabs are shared by all users ...

Reply:
Ahh right, haven't seen the header. Nice!